packages/tradershub/src/features/cfd/modals/ButtonGroups/CreateAccountButton.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Provider } from '@deriv/library';
+import { Button } from '@deriv-com/ui';
+import useDxtradeAccountHandler from '../../../../hooks/useDxtradeAccountHandler';
+import useMT5AccountHandler from '../../../../hooks/useMT5AccountHandler';
+import { validPassword } from '../../../../utils/password';
+import { MarketType, PlatformDetails, TTM5FilterLandingCompany } from '../../constants';
+
+type TCreateAccountButtonProps = {
+    buttonText: string;
+    password: string;
+    selectedJurisdiction?: TTM5FilterLandingCompany;
+};
+
+const DxtradeAccountButton = ({ buttonText, password }: TCreateAccountButtonProps) => {
+    const { createDxtradeAccountLoading, handleSubmit } = useDxtradeAccountHandler();
```

Comment:
You can omit line#17 because you are not using createDxtradeAccountLoading anywhere else
```suggestion
    const { createDxtradeAccountLoading: isLoading, handleSubmit } = useDxtradeAccountHandler();
```